README.md

```diff
@@ -23,9 +23,15 @@ In order to compile `CCL` you need a few libraries:
 * The [SWIG](http://www.swig.org/) Python wrapper generator is not needed to run `CCL`, but must be installed if you intend to modify `CCL` in any way.
 * [FFTW3](http://www.fftw.org/) is required for computation of correlation functions.
 * FFTlog([here](http://casa.colorado.edu/~ajsh/FFTLog/) and [here](https://github.com/slosar/FFTLog))is provided within `CCL`, with minor modifications.
+* The C library associated to the CLASS code. The installation of this library is described below.
+
+## Installing CLASS
+CCL uses CLASS as one of the possible ways of computing the matter power spectrum. In order to communicate with CLASS, CCL must be linked to its library. Before installing CCL proper you must therefore install this library first. Since this process is not necessarily straightforward, we provide a python script `class_install.py` that automatically downloads and install the latest tagged stable version of CLASS. You should run this script (`python class_install.py`) before carrying out the next steps. By default, the script assumes that your main C compiler is `gcc`. If that's not the case, pass the name of your C compiler to the script via the command-line argument `--c_comp` (i.e. `python class_install.py --c_comp=[name of compiler]`). Type `python class_install.py -h` for further details.
```

Comment:
I think this script downloads a specific version of CLASS - v2.6.3. Do we want to give instructions to the user to potentially replace that with a newer version when available?

Reply:
I'd say we should be in charge of keeping that up to date. If they want their own version of CLASS they should follow the steps for the custom CLASS install.

Reply:
Ok

Reply:
Should the section "Compiling against an external version of CLASS" be omitted or revised?

Reply:
Not really. These modifications automatically download and install a tagged version of CLASS that we have chosen. If users want to use any other CLASS (e.g. hi-CLASS), they have to follow the instructions in that section

Reply:
It still says "CCL has a built-in version of CLASS" (the first sentence of that section).  I guess I'm missing something, but I thought the built-in version has been removed.

Reply:
Oh, I see. I'll fix that quickly. Can you review a quick PR?